### ISSUE TITLE
How should I preprocess the dataset

### ISSUE THREAD

**Issue by Gentaiman777** [Kid-Liet/IMSE#2]
Thank you very much for your research. I have read your paper published on CVPR and I would like to run this code.

I obtained the BraTS dataset, but this is not the same as the input of the code. I want to know how to preprocess the BraTS dataset. The BraTS2019 dataset does not appear to be available at this time. The BraTS2022 dataset I obtained only has 140 training data and 20 validation data, so there is a lack of testing data and label. And how should I convert the .nii file with a dataset of 240 * 240 * 155 into the 128 * 128 * 48 npy file required in the paper. Can you provide the preprocessing code for the BraTS dataset？

**Comment by fycheng2022** [comment]
I also want to know how to prepare data.

**Comment by kuailexiaohunzi** [comment]
> Thank you very much for your research. I have read your paper published on CVPR and I would like to run this code.
> 
> I obtained the BraTS dataset, but this is not the same as the input of the code. I want to know how to preprocess the BraTS dataset. The BraTS2019 dataset does not appear to be available at this time. The BraTS2022 dataset I obtained only has 140 training data and 20 validation data, so there is a lack of testing data and label. And how should I convert the .nii file with a dataset of 240 * 240 * 155 into the 128 * 128 * 48 npy file required in the paper. Can you provide the preprocessing code for the BraTS dataset？

May I ask if the problem has been solved? I also want to know the answer
